62.CAD/main.cpp

```diff
@@ -1944,6 +1939,27 @@ class CADApp : public ApplicationBase
 			intendedNextSubmit = currentDrawBuffers.drawPolyline(polyline2, style2, submissionQueue, submissionFence, intendedNextSubmit);
 			// intendedNextSubmit = currentDrawBuffers.drawPolyline(polyline, style2, submissionQueue, submissionFence, intendedNextSubmit);
 		}
+		else if (mode == ExampleMode::CASE_4)
+		{
+			LineStyle style = {};
```

Comment:
see your TODO[Przemek] comments in main.cpp
The stipple pattern user want will go into this LineStyle struct, use some default value for hinting there is no stippling applied to decide whether to use the normal or stippled version of the SDF in the shader.